Makefile

```diff
@@ -802,17 +827,24 @@ app: appclean version data/osx/AppIcon.icns $(APPTARGET)
 	cp -R data/motd $(APPDATADIR)
 	cp -R data/credits $(APPDATADIR)
 	cp -R data/title $(APPDATADIR)
-	# bundle libc++ to fix bad buggy version on osx 10.7
-	LIBCPP=$$(otool -L $(APPTARGET) | grep libc++ | sed -n 's/\(.*\.dylib\).*/\1/p') && cp $$LIBCPP $(APPRESOURCESDIR)/ && cp $$(otool -L $$LIBCPP | grep libc++abi | sed -n 's/\(.*\.dylib\).*/\1/p') $(APPRESOURCESDIR)/
 ifdef LANGUAGES
-	ditto lang/mo $(APPRESOURCESDIR)/lang/mo
+	mkdir -p $(APPRESOURCESDIR)/lang/mo/
+	cp -pR lang/mo/ $(APPRESOURCESDIR)/lang/mo/
 endif
 ifeq ($(LOCALIZE), 1)
+ifdef OSXCROSS
+	cp $(LIBSDIR)/gettext/lib/libintl.8.dylib $(APPRESOURCESDIR)
+else
 	LIBINTL=$$(otool -L $(APPTARGET) | grep libintl | sed -n 's/\(.*\.dylib\).*/\1/p') && cp $$LIBINTL $(APPRESOURCESDIR)/
 endif
+endif
 ifdef LUA
 	cp -R lua $(APPRESOURCESDIR)/
+ifdef OSXCROSS
+	cp $(LIBSDIR)/lua/lib/liblua.5.2.4.dylib $(APPRESOURCESDIR)
```

Comment:
Or rather only supply one library file per library directory. Like:
```
.
..
liblua.5.2.4.dylib
```

instead of:

```
.
..
liblua.5.2.4.dylib
liblua.5.2.dylib -> liblua.5.2.4.dylib
liblua.dylib -> liblua.5.2.dylib
```

Then it can be easily matched by asterisk placeholder to copy whatever version is there.